protocols/googledrive.md

```diff
@@ -22,9 +22,11 @@ Using prior versions to 4.12.0 (Mountain Duck) or 8.4.0 (Cyberduck), you might *
 
 Google Drive uses OAuth 2 to allow Cyberduck to access your files on Google Drive. Choose your account email as the username in the bookmark and choose *Allow* on the website opened in your default web browser to allow Cyberduck to *View and manage the files in your Google Drive*. The authorization code will be submitted to Cyberduck automatically. Subsequent connections will not require authorization, unless the refresh token itself is expired due to inactivity.
 
-```{attention}
-Using *[Advanced Protection Program](https://support.google.com/accounts/answer/7539956#non-goog_apps&zippy=%2Ccan-i-use-non-google-apps-services-or-apps-script-with-advanced-protection)* will cause the OAuth login flow to fail. 
-```
+#### Advanced Protection Program
+
+Using *[Advanced Protection Program](https://support.google.com/accounts/answer/7539956#non-goog_apps&zippy=%2Ccan-i-use-non-google-apps-services-or-apps-script-with-advanced-protection)* will cause the OAuth login flow to fail. Mostly with the error message: `400 admin_policy_enforced`.
```

Comment:
```
Using *[Advanced Protection Program](https://support.google.com/accounts/answer/7539956#non-goog_apps&zippy=%2Ccan-i-use-non-google-apps-services-or-apps-script-with-advanced-protection)* will cause the OAuth login flow to fail with the error message `400 admin_policy_enforced`.
```

Reply:
https://support.google.com/accounts/answer/12917337?hl=en#400admin&zippy=%2Cadmin-policy-enforced

Based on my understanding the Admin can choose to allow or deny log in attempts by third party applications.